### PR TITLE
Opsexp 1447 alfcommon refactored

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-1447

* split alfresco-common to a separate chart (holding common external dependencies)
* refactor builds to allow releasing multiple dependent charts

Hope it clarifies things a little:

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+name: publish
+on:
+  workflow_run:
+    workflows:
+      - helm
+      - integration-tests
```
Comment (gionn) [inline .github/workflows/publish.yml:7]: this will trigger one time for each

File: .github/workflows/compose.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (kklassa) [inline .github/workflows/compose.yml:1]: I would suggest changing the name of this file as vs code thinks it is a docker-compose file and messes up linting

File: .github/actions/publish-charts/action.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+name: "Publish helm chart on GitHub "
+description: "Publish new version of helm chart and update repository tags"
+inputs:
+  chart:
+    description: Name of the chart to publish
+    required: true
+    type: true
```
Comment (kklassa) [inline .github/actions/publish-charts/action.yml:8]: I think this does not belong here 😅

File: .github/workflows/publish.yml (hunk 1 above)
Comment (alxgomz) [reply .github/workflows/publish.yml:7]: Yes that's expected. One publish stage for libraries charts (first), then one for for application charts (second, as they most likely depend on library)

**Pull request closed without merging**